Description: Disable parse_5 test for i386
 Due to floating point operations' unsoundness on i386 in the absence of SSE2,
 the parse_5 test fails: -2.3205080756887733 != -2.3205080756887746, which
 should be a negligible difference.
Author: NoisyCoil <noisycoil@tutanota.com>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/transform.rs
+++ b/src/transform.rs
@@ -310,6 +310,7 @@
         "matrix(0.8660254037844387 0.49999999999999994 -0.49999999999999994 0.8660254037844387 0 0)"
     );
 
+    #[cfg(any(not(target_arch = "x86"), target_feature = "sse2"))]
     test!(parse_5,
         "rotate(30 10 20)",
         "matrix(0.8660254037844387 0.49999999999999994 -0.49999999999999994 0.8660254037844387 11.339745962155611 -2.3205080756887746)"
